cg/meta/workflow/balsamic.py

```diff
@@ -278,6 +279,12 @@ def get_verified_sex(sample_data: dict) -> Sex:
             LOG.error(f"Unable to retrieve a valid sex from samples: {sample_data.keys()}")
             raise BalsamicStartError
 
+    def get_exome_argument_if_exome_sample(self, case_id: str) -> bool:
+        """Returns the exome argument if the application type in sample_data is wes."""
```

Comment:
Returns bool, please update! Or just omit doc str if function name is clear enough.

Is this independent of workflow? Seems like a general method for any workflow. Maybe move to Analysis API instead, to keep it DRY.

Reply:
I have moved the function, and the function that gets application_type for the samples within a case, which also seems like something that could be on the analysis_api level, such as for families in the rare-disease case. Just need to verify that this all still works.

Reply:
the tests I ran earlier still hold up 👍